x/staging/subspaces/keeper/keeper.go

```diff
@@ -0,0 +1,176 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/desmos-labs/desmos/x/staging/subspaces/types"
+)
+
+type Keeper struct {
+	storeKey sdk.StoreKey
+	cdc      codec.BinaryMarshaler
+}
+
+// NewKeeper creates new instances of the subspaces keeper
+func NewKeeper(storeKey sdk.StoreKey, cdc codec.BinaryMarshaler) Keeper {
+	return Keeper{
+		storeKey: storeKey,
+		cdc:      cdc,
+	}
+}
+
+// SaveSubspace saves the given subspace inside the current context.
+// It assumes that the subspace has been validated already.
+func (k Keeper) SaveSubspace(ctx sdk.Context, subspace types.Subspace) {
+	store := ctx.KVStore(k.storeKey)
+	key := types.SubspaceStoreKey(subspace.ID)
+	store.Set(key, k.cdc.MustMarshalBinaryBare(&subspace))
+}
+
+// DoesSubspaceExist checks if the subspace with the given id exists.
+func (k Keeper) DoesSubspaceExist(ctx sdk.Context, subspaceID string) bool {
+	store := ctx.KVStore(k.storeKey)
+	return store.Has(types.SubspaceStoreKey(subspaceID))
+}
+
+// GetSubspace returns the subspace associated with the given ID.
+// If there is no subspace associated with the given ID the function will return an empty subspace and false.
+func (k Keeper) GetSubspace(ctx sdk.Context, subspaceID string) (subspace types.Subspace, found bool) {
+	if !k.DoesSubspaceExist(ctx, subspaceID) {
+		return types.Subspace{}, false
+	}
+
+	store := ctx.KVStore(k.storeKey)
+	k.cdc.MustUnmarshalBinaryBare(store.Get(types.SubspaceStoreKey(subspaceID)), &subspace)
+	return subspace, true
+}
+
+// GetAllSubspaces returns a list of all the subspaces that have been store inside the given context
+func (k Keeper) GetAllSubspaces(ctx sdk.Context) []types.Subspace {
```

Comment:
Can we move this to a new file named `alias_functions.go`?

Reply:
I will rename `common_functions` to `alias_functions` and move it there